openedx_filters/learning/filters.py

```diff
@@ -701,3 +701,37 @@ def run_filter(cls, context, template_name):
         """
         data = super().run_pipeline(context=context, template_name=template_name)
         return data.get("context"), data.get("template_name")
+
+
+class ORASubmissionViewRenderStarted(OpenEdxPublicFilter):
+    """
+    Custom class used to create ORA submission view filters and its custom methods.
+    """
+
+    filter_type = "org.openedx.learning.ora.submission_view.render.started.v1"
+
+    class RenderInvalidTemplate(OpenEdxFilterException):
+        """
+        Custom class used to stop the dashboard render process.
+        """
+
+        def __init__(self, message: str, template: str = ""):
+            """
+            Override init that defines specific arguments used in the submission view render process.
+
+            Arguments:
+                message (str): error message for the exception.
+                template (str): template path rendered instead.
+            """
+            super().__init__(message, template=template)
+
+    @classmethod
+    def run_filter(cls, template_name: str):
+        """
+        Execute a filter with the signature specified.
+
+        Arguments:
+            template_name (str): template name to be rendered by the student's dashboard.
+        """
+        data = super().run_pipeline(template_name=template_name)
+        return data.get("template_name")
```

Comment:
why was the context field removed? Usually, for backend renders there's a context to be sent to the template. Why is that not the case here?

Reply:
In this case, we don't make use of the context, we only want to modify the template, but we could add it.

Reply:
Finally, I added the context to the filter